packages/vitest/src/runtime/collect.ts

```diff
@@ -31,7 +35,13 @@ export async function collectTests(paths: string[], config: ResolvedConfig) {
     clearContext()
     try {
       await runSetupFiles(config)
-      await import(filepath)
+      if (typeof window === 'undefined') {
+        await import(filepath)
+      }
+      else {
+        // fake cache for browser
+        await import(`${filepath}?v=${new Date().getTime()}`)
```

Comment:
we need to support windows, adding this will work:
```ts
        // fake cache for browser and windows fix: X:/<path> => /@fs/<path>
        const math = filepath.match(/^(\w:\/)/)
        if (math)
          await import(`/@fs/${filepath.slice(math[1].length)}?v=${new Date().getTime()}`)
        else
          await import(`${filepath}?v=${new Date().getTime()}`)
```